molecule/default/group_vars/all.yml

```diff
@@ -4,9 +4,11 @@
 
 timescaledb_pg_major: 12
 timescaledb_pg_user: "postgres"
+timescaledb_pg_group: "postgres"
```

Comment:
i think, line 5 to 7 are not needed here, since they are the same variables that is already defined in Defaults.

Reply:
Well spotted :) . Removed